src/style.css

```
@@ -0,0 +1,100 @@
+/* fonts */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+/* global */
+
+* {
+  box-sizing: border-box;
+}
+
+/* intent between bars */
+
+.first__form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
```

don't add `first` classes for `nth` elements, in this exact case you can just create base `form` class like this
```suggestion
.form {
  margin-top: 20px;
}
```